baseStats.json

```diff
@@ -321,13 +321,12 @@
       "evolvesTo": 0,
       "evolvesFrom": "Kakuna",
       "quickMoves": [
-        "Bug Bite",
-        "Poison Jab"
+        201,
+        224
       ],
       "cinematicMoves": [
-        "Sludge Bomb",
-        "Aerial Ace",
-        "X Scissor"
+        90,
+        45
```

Comment:
Well, in this line it was because of X Scissor, but in the other lines I pointed out the missing attacks are Seed Bomb, Solar Beam, Bug Buzz and Submission.